docs/reference_checklist.md

```diff
@@ -73,11 +74,11 @@ After implementing the op:
     1. For precision mismatches, tag the test with `RUN-DISABLED(#1278)` (if
        it's not already done).
 1. In [ops_stablehlo.mlir](https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/ops_stablehlo.mlir):
-    1. Make sure that there is at least one test for each constraint in the
-       verifier and type inference methods; constraints covered in ODS will not
-       be tested. These tests will mostly be negative tests exercising the fact
-       that the constraints are not met. Make sure to add at least one positive
-       test in case there are none.
+    1. Make sure that there is at least one test (positive or negative) for each
+       constraint in the verifier and type inference methods; constraints
+       covered in ODS will not be tested. These tests will mostly be negative,
+       testing that the constraints are not met or positive, testing that the
+       inferred shape is correct.
```

Comment:
Just to document that we are not adding negative tests with invalid result type, where the error is thrown from the type inference code.

Reply:
This is a "no action". Just a comment.